Tasks/Common/sonarqube-common/sonarqube-common.ts

```diff
@@ -87,6 +90,28 @@ export function getSonarQubeEndpointFromInput(inputFieldName): SonarQubeEndpoint
     return new SonarQubeEndpoint(hostUrl, hostUsername, hostPassword);
 }
 
+// Returns, as an object, the contents of the 'report-task.txt' file created by SonarQube plugins
+// The returned object contains the following properties:
+//   projectKey, serverUrl, dashboardUrl, ceTaskId, ceTaskUrl
+export function getSonarQubeTaskReport(sonarPluginFolder: string) {
+    var reportFilePath:string = path.join(sonarPluginFolder, 'report-task.txt');
+    if (!tl.exist(reportFilePath)) {
+        tl.debug('Task report not found at: ' + reportFilePath);
```

Comment:
The precursor to this function, findCandidateModules, has a high rate of false positives (especially if there is more than one code analysis tool running, i.e. PMD), so this is not necessarily a problem.

I will, however, add a warning for if no task reports are found at all.

Accepted, in progress.